Mapping/Loader/AnnotationLoader.php

```diff
@@ -112,7 +116,7 @@ public function getMetaData($entity, EntityManager $em)
         if (!$description->isEmpty()) {
 
             $meta = $em->getClassMetadata($class);
-            $description->setIdentifierAttributeNames($meta->getIdentifierFieldNames());
+            $description->mergeIdentifierAttributeNames($meta->getIdentifierFieldNames());
```

Comment:
I guess if you have one or more custom ```Id``` annotation, you do not want the default one from symfony